frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -77,15 +77,16 @@ const styles = {
 
 class EditEmail extends Component {
   static propTypes = {
-    onChange: PropTypes.func.isRequired
+    onChange: PropTypes.func.isRequired,
+    action: actionShape
   };
 
   state = {
-    emailType: EMAIL_TYPE.reply,
-    emailTo: "",
-    emailCc: "",
-    emailBody: "",
-    reasonsForAction: ""
+    emailType: this.props.action == null ? EMAIL_TYPE.reply : this.props.action.emailType,
```

Comment:
instead of `this.props.action == null`, you can just say `!this.props.action`

https://stackoverflow.com/questions/6003884/how-do-i-check-for-null-values-in-javascript/6003920

Reply:
Fixed in b2e0b61